pkg/operator/starter.go

```diff
@@ -127,6 +128,22 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		controllerConfig.EventRecorder,
 	)
 
+	if cache.WaitForCacheSync(ctx.Done(), configInformers.Config().V1().ClusterVersions().Informer().HasSynced) {
+		return fmt.Errorf("unable to sync ClusterVersions informer")
+	}
+	cv, err := configInformers.Config().V1().ClusterVersions().Lister().Get("version")
+	if err != nil {
+		return err
+	}
+
+	buildEnabled := false
+	for _, capability := range cv.Status.Capabilities.EnabledCapabilities {
+		if capability == configv1.ClusterVersionCapabilityBuild {
```

Comment:
While cluster capabilities [cannot be disabled][1], additional cluster capabilities [can be enabled post-install][2].   So checking once at start-up time is ok, but you probably want some kind of system to trigger a container restart, or informer re-configure, or similar if `Build` is disabled when you started up but gets enabled later.  As you have it now, this controller won't notice the change until the container restarts for some other reason, and in a stable cluster that might be a long time after `Build` was enabled in ClusterVersion.

[1]: https://github.com/openshift/enhancements/blob/7af16a6363d0325d3756d2892b85bbd213b1860d/enhancements/installer/component-selection.md#capabilities-cannot-be-uninstalled
[2]: https://github.com/openshift/enhancements/blob/7af16a6363d0325d3756d2892b85bbd213b1860d/enhancements/installer/component-selection.md#capabilities-can-be-installed